### DISCUSSION TITLE
RichTextBlock Margin and Border Issue.

### DISCUSSION THREAD

**Discussion by ward-horsfall** [unoplatform/uno#18939] [Q&A] [answered]
Hi,

I have a RichTextBlock and I have two issues with the output.

Margin:

It only seems to work properly with the Left and Top. The right and bottom seem to be ignored.
The idea I am going for a large volume of text it scrolls within the defined area and does not
go over the margins.

Scroll Bars:

I want to have the scroll bars always visible and thick like seen in the image. Either the scroll bar
is not seen or two thin. (I want them to look like the vertical one in the first image).

How can I change my XAML to fix this?

See my code in: MainPage_xaml.txt

Thanks

Ward

![Border_Issue](https://github.com/user-attachments/assets/6e88081a-1047-43fc-8fb7-85fa4ed5bed9)
![Border_Issue_No_Scrollbar](https://github.com/user-attachments/assets/ff6b8965-b618-480b-b7ef-7cdfb63e2885)
[MainPage_xaml.txt](https://github.com/user-attachments/files/17931625/MainPage_xaml.txt)

**Comment by jeromelaban** [comment, marked as answer]
Thanks for the report! As you're looking at the WinAppSDK target framework and not any of the uno targets (everything by `netX.0-windows10.YYY`), it's generally best to ask questions directly at https://github.com/microsoft/microsoft-ui-xaml.

Note that Uno does not yet support RichTextBlock.

**Reply by ward-horsfall** [reply]
Hi https://github.com/jeromelaban,

Thanks for your response. Good idea - in error I assumed UNO supports all the controls :)

Thanks again,

Ward
